examples/impute/plot_iterative_imputer_variants_comparison.py

```diff
@@ -0,0 +1,126 @@
+"""
+=========================================================
+Imputing missing values with variants of IterativeImputer
+=========================================================
+
+The :class:`sklearn.impute.IterativeImputer` class is very flexible - it can be
+used with a variety of predictors to do round-robin regression, treating every
+variable as an output in turn.
+
+In this example we compare some predictors for the purpose of missing feature
+imputation with :class:`sklearn.imputeIterativeImputer`::
+
+    :class:`sklearn.linear_model.BayesianRidge`: regularized linear regression
```

Comment:
Is there a reason for this ordering?

Reply:
No. Should there be?

Reply:
I think it would be better if there was. At least it seems awkward that the tree and the forest are not together if it's not alphabetical. Make it alphabetical by class name. Hide the import path by putting a ~ after the first `.

Reply:
Done.